src/metatags/handler.js

```diff
@@ -108,9 +108,9 @@ export async function fetchAndProcessPageObject(s3Client, bucketName, url, key,
   // handling for homepage
   return {
     [pageUrl]: {
-      title: object.scrapeResult.tags.title,
-      description: object.scrapeResult.tags.description,
-      h1: object.scrapeResult.tags.h1 || [],
+      title: trimTagValue(object.scrapeResult.tags.title),
+      description: trimTagValue(object.scrapeResult.tags.description),
+      h1: trimTagValue(object.scrapeResult.tags.h1) || [],
```

Comment:
Looks like fix should be
`h1: (object.scrapeResult.tags.h1 || []).map(h => trimTagValue(h)),`

because
object.scrapeResult.tags.h1 is `list`, but it isn't a `string`